views/components/tabular-data.html

```diff
@@ -0,0 +1,29 @@
+{% macro tabularData(myLatest) %}
+<article class="base--article" itemscope itemtype="http://schema.org/Table">
+  <header class="base--header">
+    <h1 class="base--h1" itemprop="name">{{myLatest.title}}</h1>
+  </header>
+  <table class="base--table">
+    {% if myLatest.caption %}
+    <caption class="base--caption" itemprop="caption" itemscope itemtype="https://schema.org/Property">{{myLatest.caption}}</caption>
+    {% endif %}
+    <thead class="base--thead">
+      <tr class="base--tr">
+        {% for tableHeader in myLatest.tableHeaders %}
+        <th class="base--th">{{tableHeader}}</th>
+        {% endfor %}
+      </tr>
+    </thead>
+    <tbody class="base--tbody">
+        {% for table in myLatest.table %}
+        <tr class="base--tr">
+          <td data-th="{{table.headers[0]}}" class="base--td" itemprop="name" itemscope itemtype="https://schema.org/Table">{{table.identifier}}</td>
```

Comment:
Tried this, but the `{{row[loop.index]}}` didn't work for me and I think how I have the `header` data setup that wouldn't' work either. Basically, I couldn't get the `headers` to show up here.